### DISCUSSION TITLE
Material X support?

### DISCUSSION THREAD

**Discussion by Helio-Alex** [dreamworksanimation/openmoonray#157] [Q&A] [answered]
Hello,

I was wondering if MaterialX through USD is going to be something that will be considered in future versions?

Thank you!

**Comment by jlanz** [comment, marked as answer]
Hi, thanks for your interest!
We announced yesterday at the ASWF Open Source Days BOF that we have begun initial work on MaterialX support through USD.  The current goal is to support the standard MaterialX nodes as first class MoonRay shaders.  This work is early and we don't have an anticipated completion date at this time, but we are making good progress!

**Reply by Helio-Alex** [reply]
Hello! Thank you for your reply! 

I am looking forward to testing it when it's ready.
